open_schools_platform/organization_management/circles/models.py

```diff
@@ -46,9 +50,20 @@ def latitude(self):
     def longitude(self):
         return self.location.x
 
+    @property
+    def student_profile_queries(self):
+        from open_schools_platform.student_management.students.models import StudentProfile
+        from open_schools_platform.student_management.students.models import Student
+
+        return self.recipient_queries.all().filter(
+            sender_ct=ContentType.objects.get_for_model(StudentProfile),
+            body_ct=ContentType.objects.get_for_model(Student)
+        )
+
     def __str__(self):
         return self.name
 
+    # TODO: locate this logic to service layer without local import?
```

Comment:
Just move to services won't work. But yes, we need to resolve all local imports in project